### PR TITLE
Fix two links in CHANGELOG

### PR DESCRIPTION
**Pull request by coopergillan**
## Description

Update the text of two links to match the actual link.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,13 +3,13 @@
 ## Unreleased
 
 ## v2.8.2
-* Handles ErroneousStateError, in previous versions the consumer would do several unecessary group leave/joins. The log level is also changed to WARN instead of ERROR. ([#285](https://github.com/zendesk/racecar/pull/295))
+* Handles ErroneousStateError, in previous versions the consumer would do several unecessary group leave/joins. The log level is also changed to WARN instead of ERROR. ([#295](https://github.com/zendesk/racecar/pull/295))
```
Comment (coopergillan) [inline CHANGELOG.md:6]: Looking at both #285 and #295, it looks like this should definitely be #295.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
 
 ## v2.8.1
 * Adds new ErroneousStateError to racecar in order to give more information on this new possible exception.
 
 ## v2.8.0
-* Update librdkafka version from 1.8.2 to 1.9.0 by upgrading from rdkafka 0.10.0 to 0.12.0. ([#283](https://github.com/zendesk/racecar/pull/293))
+* Update librdkafka version from 1.8.2 to 1.9.0 by upgrading from rdkafka 0.10.0 to 0.12.0. ([#293](https://github.com/zendesk/racecar/pull/293))
```
Comment (coopergillan) [inline CHANGELOG.md:12]: Same here: looking at the description, this should definitely be #293.

**Comment by coopergillan** [comment]
@leonmaia - does this change look good to you? Looks like the builds won't run unless someone kicks them off.

**Comment by coopergillan** [comment]
@darena-zendesk - are you able to kick off these specs?

**Review by deepredsky — APPROVED** [review]
Thanks for fixing the links 👍

**Comment by deepredsky** [comment]
@coopergillan if you rebase/recommit this should trigger the CI

**Review by jbdietrich — APPROVED** [review]

**Review by jbdietrich — DISMISSED** [review]

**Review by jbdietrich — APPROVED** [review]

**Pull request merged**